elementary/clients/dbt/dbt_runner.py

```diff
@@ -11,7 +11,18 @@
 from elementary.utils.log import get_logger
 
 logger = get_logger(__name__)
+import yaml
 
+_MONITOR_DIR = os.path.dirname(os.path.realpath(__file__))
+
+_PACKAGES_FILENAME = "packages.yml"
+
+PATH = os.path.join(_MONITOR_DIR, "dbt_project")
```

Comment:
Actually this does not make sense because it couples `DbtRunner` to the internal dbt project while it should remain generic. Instead, we should use `self.project_dir` and join with it everywhere.
Sorry for misleading you 🙏